client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -45,20 +66,24 @@ const StatsNavigation = ( props ) => {
 					{ sectionTitles.year }
 				</NavItem>
 			</NavTabs>
+			{ statsControl }
 			<FollowersCount />
 		</SectionNav>
 	);
 };
 
 StatsNavigation.propTypes = {
+	isJetpack: PropTypes.bool,
+	isWooConnect: PropTypes.bool,
 	section: PropTypes.string.isRequired,
 	slug: PropTypes.string,
+	siteId: PropTypes.number,
 };
 
 const connectComponent = connect(
-	state => {
+	( state, props ) => {
 		return {
-			slug: getSelectedSiteSlug( state )
+			isWooConnect: props.isJetpack && hasActivePlugin( state, props.siteId, 'woocommerce' )
```

Comment:
Your assumption is correct. I included the check of `isJetpack` so it would read more logically. I could include a check for jetpack inside the `hasActivePlugin`. I could also eliminate it altogether (as you pointed out), but then I'd be relying on the implicit knowledge of the relationship between jetpack, plugins, and wpcom.